### PR TITLE
Add csv files in the test folder

### LINKED ISSUE #111: Add the Norse CSV files to the repo test folder

### PR DESCRIPTION
**Pull request by gnawin**
# Pull request details

## Describe the changes made in this pull request

Add csv files for the Norse case in the test folder

## List of related issues or pull requests

Closes #111 

## Collaboration confirmation

As a contributor I confirm

- [x] I read and followed the instructions in README.dev.md
- [x] The documentation is up to date with the changes introduced in this Pull Request (or NA)
- [x] Tests are passing
- [x] Lint is passing

### CODE REVIEW

File: test/inputs/Norse/rep-periods-data.csv
Comment (datejada) [inline test/inputs/Norse/rep-periods-data.csv:1]: The representative weight is not correct. It should be only one representative with a weight of 52.1423 hours

**Review by datejada — CHANGES_REQUESTED** [review]
Change the rep-periods-data

File: test/inputs/Norse/rep-periods-data.csv
Comment (gnawin) [reply test/inputs/Norse/rep-periods-data.csv:1]: It is updated.

**Review by datejada — APPROVED** [review]
Thanks for the change. It looks good for me.

**Pull request merged**